images/installer/Dockerfile.ci

```diff
@@ -6,7 +6,7 @@ WORKDIR /go/src/github.com/openshift/installer
 COPY . .
 RUN hack/build.sh && hack/get-terraform.sh
 
-FROM scratch
+FROM centos:7
```

Comment:
This is [effectively what's happening in production][1], although it's "whatever the OpenShift base layer is" their and not explicitly "CentOS 7".  The image I was building for testing openshift/release#1677 is `registry.svc.ci.openshift.org/wking-next-gen-installer/stable:installer` in case you want to poke around (I think that is public?).   The policy doc has:

> Do not ask to have your image based of scratch, busybox, alpine, etc. The request will be denied.

I've dropped an internal comment asking why they forbid `scratch`, and until then, meh, I'm fine either way ;).

[1]: https://github.com/openshift/release/pull/1677#discussion_r220890891